packages/components/src/components/primitives/Button/types.ts

```diff
@@ -0,0 +1,9 @@
+import { ButtonProps as ChakraButtonProps } from '@chakra-ui/react';
+import { Sizes } from '../../../theme/types';
+
+export const ButtonVariants = ['primary', 'secondary', 'tertiary'] as const;
+export type VariantType = typeof ButtonVariants[number];
```

Comment:
VariantType is generic and this is exported to all the library, please rename to `ButtonVariant`

Reply:
renamed